assets/quick-start-module.js

```diff
@@ -267,7 +267,7 @@ $("[data-toggle='cloud-dropdown']").on("click", function(e) {
 });
 
 function commandMessage(key) {
```

Comment:
This does not look correct. Update to quickstart is automated: https://github.com/pytorch/pytorch.github.io/blob/site/.github/workflows/update-quick-start-module.yml#L93